ide/git/src/org/netbeans/modules/git/ui/checkout/SwitchBranchAction.java

```diff
@@ -47,6 +47,17 @@
 @ActionRegistration(displayName = "#LBL_SwitchBranchAction_Name")
 public class SwitchBranchAction extends AbstractCheckoutAction {
 
+    private static final String ICON_RESOURCE = "org/netbeans/modules/git/resources/icons/active_branch.png"; //NOI18N
```

Comment:
used `active_branch.png` for the switch branch action since I didn't find a better option